### PR TITLE
Module I:Capstone Project

### PR DESCRIPTION
**Pull request by Kevin-Mena**
**In this project,I:**
✔ Created a homepage of the website
✔ Created about page of the  website
✔ Added markup to both the homepage and about page
✔ Added styling to both the homepage and about page
✔ Added dynamics to the pages with Javascript

### CODE REVIEW

**Comment by NduatiKagiri** [comment]
Hi @Kevin-Mena 👋 , 

# INVALID REVIEW REQUEST ❌ ❌ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not documented. Kindly document your project using this [README template.](https://github.com/microverseinc/readme-template).
Please also remember to create the video requested in the requirements and submit the link to the video in the PR description.
Kindly make sure that you have deployed your project correctly because it doesn't seem to work.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -616,3 +616,284 @@ section featured speakers
...
 }
 
+/*
+ ===================================
+****|| DESKTOP VERSION  MEDIA QUERIES ||****
+=================================== */
+@media screen and (min-width: 768px) {
+  .btn-toggle {
+    display: none;
+  }
```
Comment (andgarzonmal) [inline styles.css]: - [ ] Kindly follow the styles of template as much as you can 

## I made  this videos to show what you need to correct please take a look at them 

####  https://www.loom.com/share/37ebe5c1cd1f40678db9620f55a1389c
#### https://www.loom.com/share/1fbc538318804d438f4ef4ac26d0013f

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @Kevin-Mena ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- You set up linter files and no errors. ✔️


# ♻️ Request changes ♻️
# ‼️ important  ‼️
- [ ]  Please add the loom video link to the description of your pull request

- [ ] I noticed you added the readme file on the main branch and it looks professional, but please also add it on this branch.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: styles.css (hunk 1 above)
Comment (Kevin-Mena) [reply styles.css:626]: Thank you @andgarzonmal  for the loom videos,they were of a great help to me

File: index.html [hunk 2]
```diff
@@ -114,7 +114,7 @@
    <section class="intro" id="home">
      <div class="main-content">
         <p class="quote">&#34;hello world!welcome to&#34;</p>
-        <h2 class="header-content">chat<span class="gpt">gpt</span> forum: fast coding.the future of software development</h2>
+        <h1 class="header-content">chatgpt annual global forum<br> 2023</h1>
         <p class="descr">A joyful celebration of the succesful launch of the ChatGPT(Chat Generative Pre-trained Transformer) model.Meet developers from over 105 countries during the celebration in San Franscisco,California </p> 
         <p class="date">2023.12.11(TUE)~(WED)</p>  
         <p class="address">&#169;AMA Conference Center San Francisco and more</p>
```
Comment (mwanawabangona) [inline index.html:120]: - [ ] `required`
- [ ] Amazing job so far 👍 
- [ ] however, there is a few things you to put in place.
- [ ] please follow the fonts size as close as possible
- [ ] also i suggest you add the border to the paragraph
- [ ] please see screenshots below

`your template`
image

`Cindy shin template`
image

File: index.html [hunk 3]
```diff
@@ -209,54 +209,53 @@ <h2 class="speaker-header">featured speakers</h2>
         <button class="more">
            MORE <span class="fa-more"><i class="fa-solid fa-angle-down"></i></span>
         </button>
-    </div>
-    
+    </div> 
   </section>
- <section class="forum-partners">
```
Comment (mwanawabangona) [inline index.html:215]: - [ ] `required`
- [ ] please add the required background
- [ ] please see screenshot below

`your template`
image

`cindy shin template`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi, @Kevin-Mena 

Great work on making the changes requested by a previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 4]
```diff
@@ -178,6 +194,7 @@ li .nav-link {
   background-color: var(--font-color-1);
   height: auto;
   display: block;
+  background-image: url(./images/carbon.avif);
```
Comment (emmiiorji) [inline styles.css]: Well done for adjusting the background image. However, it's yet to match the given design. Kindly make a copy of the image as closely as possible. You may use [this](https://user-images.githubusercontent.com/20554583/218251204-efa0057a-776f-4e31-93e0-890589a6f2ef.jpg).

Your design:
image

Design guide:
screenshot

File: styles.css [hunk 5]
```diff
@@ -607,12 +653,329 @@ section featured speakers
...
+  }
+
+  .header {
+    position: relative;
+  }
+
+  .nav-bar {
+    position: sticky;
+  }
+
```
Comment (emmiiorji) [inline styles.css:698]: [Optional] Kindly adjust the design to remove the horizontal scroll so as to improve the user experience.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Kevin-Mena, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Some of the requested changes have been implemented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Kevin-Mena,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- The background has been fixed as suggested by the previous reviewer :+1: 
- No linter errors :+1: 
## Optional suggestions :small_airplane: 


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kevin-Mena** [comment]
@GedeonTS  Long thread,thanks for approving😂😂😂

**Pull request merged**